packages/data-context/__snapshots__/codegen.spec.ts.js

```diff
@@ -7,7 +7,7 @@ module.exports = defineConfig({
     // We've imported your old cypress plugins here.
     // You may want to clean this up later by importing these.
     setupNodeEvents(on, config) {
-      return require('./')(on, config)
```

Comment:
These were all wrong, how the heck did this get merged? This is why I don't generally like snapshots; they don't test behavior.